cpp/src/parquet/properties.h

```diff
@@ -47,6 +47,13 @@ namespace parquet {
 /// DataPageV2 at all.
 enum class ParquetDataPageVersion { V1, V2 };
 
+/// Controls the level of size statistics that are written to the file.
+enum class SizeStatisticsLevel : uint8_t {
+  NONE = 0,  // No size statistics are written.
+  CHUNK,     // Only column chunk size statistics are written.
+  PAGE       // Both size statistics in the column chunk and page index are written.
+};
```

Comment:
1) Uppercase identifiers always risk colliding with C macros (especially with Windows headers)
2) "chunk" is too vague, "column chunk" would be more explicit

Therefore I would suggest:
```suggestion
enum class SizeStatisticsLevel : uint8_t {
  None = 0,    // No size statistics are written.
  ColumnChunk, // Only column chunk size statistics are written.
  Page         // Both size statistics in the column chunk and page index are written.
};
```